kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/internal/util/Metrics.java

```diff
@@ -99,6 +100,74 @@ public class Metrics {
                                                                                                                                                 .withRegistry(
                                                                                                                                                         globalRegistry);
 
+    public static final ClusterNodeSpecificMetricProviderCreator<DistributionSummary> KROXYLICIOUS_CLIENT_TO_PROXY_REQUEST_SIZE_METER_PROVIDER = (virtualClusterName,
+                                                                                                                                                  nodeId) -> DistributionSummary
+                                                                                                                                                          .builder(
+                                                                                                                                                                  KROXYLICIOUS_CLIENT_TO_PROXY_REQUEST_BASE_METER_NAME
+                                                                                                                                                                          + "_size")
+                                                                                                                                                          .baseUnit(
+                                                                                                                                                                  BaseUnits.BYTES)
+                                                                                                                                                          .description(
```

Comment:
The formatting here is awful. It might be better to use an anonymous class, or method reference, rather than a lambda.